contrib/completion/git-prompt.sh

```diff
@@ -35,6 +35,11 @@
 #
```

Comment:
> I can't reproduce this.

@ghedsouza can you? If so, maybe you can reply to the mail (you can follow [this advice](https://github.com/gitgitgadget/gitgitgadget/wiki/ReplyToThis) to reply even if you are not subscribed to the mailing list).

And even if you cannot reproduce this anymore, it would be nice to close the circle and reply to the reviewer comment.

Reply:
@dscho No, I can't reproduce it. I tried to reply to the email through gmail, but it got rejected from `git@vger.kernel.org` because it had HTML content in it.

I just tried again in plain-text mode. It looks like it worked because I see it on the mailing list mirror.

Hopefully my next patch request is little more well thought out :p

Reply:
> it got rejected from `git@vger.kernel.org` because it had HTML content in it.

Right, I tried to make sure that this is mentioned in https://github.com/gitgitgadget/gitgitgadget/wiki/ReplyToThis. Maybe you can improve that page? It's a wiki.

> I just tried again in plain-text mode. It looks like it worked because I see it on the mailing list mirror.

Yep, I got it, too (I'm subscribed).

> Hopefully my next patch request is little more well thought out :p

No worries. I'd rather err on the side of having one contribution too many than one too few.